test/t8n/t8n.cpp

```diff
@@ -156,6 +176,9 @@ int main(int argc, const char* argv[])
                         transactions.emplace_back(std::move(tx));
                         receipts.emplace_back(std::move(receipt));
                     }
+                    // Set original std::clob buffer
```

Comment:
typo.
Also this is not exception safe and probably not needed. I'd go with just leaving a comment when rdbuf is modified that we will not restore the original.

Reply:
When don't do it the program is crashing randomly.